packages/sdk/src/common/http/FetchHttpClient.ts

```
@@ -178,10 +182,65 @@ class FetchHttpClient implements HttpClient {
+                        path: httpPath.path,
+                        query: httpQuery.query,
+                        timeout: this.options.timeout
+                    },
+                    error
+                );
+            }
+            
+            // Handle other network errors
+            throw new HttpNetworkException(
+                `${FQP}get(httpPath: HttpPath, httpQuery: HttpQuery): Promise<Response>`,
+                error instanceof Error ? error.message : 'Network request failed',
```

ESLint

[error] Replace `·?·error.message` with `⏎····················?·error.message⏎···················`

(prettier/prettier)